### ISSUE TITLE
Always the same world

### ISSUE THREAD

**Issue by Jackey33** [SubnauticaNitrox/Nitrox#1258]
### Version

 1.3.0.0

### Bug
 When I try to start the server, it always has the same world generation even if I completely deleted world folder

### Other details:

 
[Player.log](https://github.com/SubnauticaNitrox/Nitrox/files/5345850/Player.log)

**Comment by Seba10** [comment]
I mean the world stays the same where items and harvestable objects spawn is different

**Comment by Jackey33** [comment]
> I mean the world stays the same where items and harvestable objects spawn is different

I mean it's completely the same, including items and landscape

**Comment by Untar1** [comment]
> I mean it's completely the same, including items and landscape

Just to clarify - the random world generation in Subnautica was removed a long time ago, as far as I remember. So the landscape cannot change either. So I assume the only issue you encountered is the item spawn being the same over several saves.

Or did the position of the Lifepod also persist?
How many worlds did you try?

**Comment by Jackey33** [comment]
> > I mean it's completely the same, including items and landscape
> 
> Just to clarify - the random world generation in Subnautica was removed a long time ago, as far as I remember. So the landscape cannot change either. So I assume the only issue you encountered is the item spawn being the same over several saves.
> 
> Or did the position of the Lifepod also persist?
> How many worlds did you try?

Yes, the position of the Lifepod is also the same, I tried over 20 times

**Comment by dartasen** [comment]
Yeah, we decided not to use the random lifepod feature to make sync easier for us.
But we definitly want to readd this feature and the ability to have a Lifepod per player !

**Comment by Jackey33** [comment]
> Yeah, we decided not to use the random lifepod feature to make sync easier for us.
> But we definitly want to readd this feature and the ability to have a Lifepod per player !

That would be cool. 
By the way, there were also some bugs like some creatures just stay still, or we couldn't make the second seamoth, so we had to use cheats, but the most disappointing bugs was position and texture bugs: 
1) I was driving through the hole at the top of the Aurora, but suddenly seamoth broke for no reason. My friend said I hit the ship hull, but actually, I was next to it.
2) When we came to Aurora, my friend went inside without any problem, but I didn't see the texture of the floor. I thought it was just a texture bug and I just can't see the floor, but when I made a step on it, I fell to the bottom and couldn't get back.
If it matters, I was the host played through the steam, and my friend through the epic games store.
